addons/mrp/tests/test_order.py

```diff
@@ -782,9 +782,6 @@ def test_product_produce_6(self):
         mo_form.qty_producing = 3
         mo = mo_form.save()
 
-        mo._post_inventory()
-        self.assertEqual(len(mo.move_raw_ids), 4)
-
         mo.move_raw_ids.filtered(lambda m: m.state != 'done')[0].quantity = 3
```

Comment:
since you're removing the `_post_inventory` the moves should never be `'done'` until the `button_mark_done` is called